torchmdnet/neighbors/neighbors_cpu.cpp

```diff
@@ -90,7 +90,9 @@ forward(const Tensor& positions, const Tensor& batch, const Tensor& box_vectors,
 }
 
 TORCH_LIBRARY_IMPL(torchmdnet_neighbors, CPU, m) {
-    m.impl("get_neighbor_pairs_brute", &forward);
-    m.impl("get_neighbor_pairs_shared", &forward);
-    m.impl("get_neighbor_pairs_cell", &forward);
+  m.impl("get_neighbor_pairs", [](std::string strategy,  const Tensor& positions, const Tensor& batch, const Tensor& box_vectors,
```

Comment:
Maybe:
```c++
const std::string strategy
```

Reply:
I was being evil and changing the value of strategy under some circumstances. I agree, lets do it the right way :P